frontend/common/crud/requestCommon.ts

```diff
@@ -5,7 +5,7 @@ export function requestCommon(options: {
   method: string;
   body?: unknown;
   headers?: HeadersInit;
-  signal: AbortSignal;
+  signal?: AbortSignal;
```

Comment:
there is never a case where we would not want an abort signal passed in here
making this optional will allow devs to have the bad practice of not supporting an abort signal

Reply:
That was a test leftover. Updated.